lua/cmp_tabnine/source.lua

```diff
@@ -143,6 +151,7 @@ function Source.complete(self, ctx, callback)
 end
 
 Source._on_err = function(_, _, _)
+	table.remove(Source.ctx_list, 1)
```

Comment:
I dont think this is needed.

Reply:
Does tn ever produce response in stderr? In that case `on_stdout` will not be called and `ctx_list` will have a stale ctx for the next call of `on_stdout`. Or I misunderstand job control of vim?

Reply:
I am not sure what are TabNine's semantics in this case, but I think if we get anything on err, we can restart the process.